lib/ui/trash_page.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:io';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:photos/core/event_bus.dart';
+import 'package:photos/db/trash_db.dart';
+import 'package:photos/events/files_updated_event.dart';
+import 'package:photos/models/file_load_result.dart';
+import 'package:photos/models/selected_files.dart';
+
+import 'gallery.dart';
+import 'gallery_app_bar_widget.dart';
+
+class TrashPage extends StatelessWidget {
+  final String tagPrefix;
+  final GalleryAppBarType appBarType;
+  final _selectedFiles = SelectedFiles();
+
+  TrashPage(
+      {this.tagPrefix = "trash_page",
+      this.appBarType = GalleryAppBarType.trash,
+      Key key})
+      : super(key: key);
+
+  @override
+  Widget build(Object context) {
+    final gallery = Gallery(
+        asyncLoader: (creationStartTime, creationEndTime, {limit, asc}) {
+          return TrashDB.instance.getTrashedFiles(
```

Comment:
We could in future query by `deleteBy` and inject a `labelRenderer(time)` that renders the days left for deletion. Not necessary now, just saying.